language-plutus-core/src/Language/PlutusCore/Evaluation/Machine/L.hs

```diff
@@ -37,6 +37,7 @@ module Language.PlutusCore.Evaluation.Machine.L
     , EvaluationResultDef
```

Comment:
As I said, I'd be happy to just delete this module.

Reply:
Let's have here the updated version and delete it in a subsequent PR. That will make reverting the deletion easier should we need the lazy machine later.